code/modules/mob/living/simple_animal/hostile/illusion.dm

```diff
@@ -58,7 +58,7 @@
 			return
 		var/mob/living/simple_animal/hostile/illusion/M = new(loc)
 		M.faction = faction.Copy()
-		M.Copy_Parent(parent_mob, life_span, health/2, melee_damage_upper, multiply_chance)
+		M.Copy_Parent(parent_mob, 80, health/2, melee_damage_upper, multiply_chance)
```

Comment:
each generation is still exponentially more mobs than the previous, which can easily kill the server when given to any mildly robust player.